packages/types/src/extrinsic/v2/Doughnut.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Plug New Zealand Limited
+//
```

Comment:
Doughnut should not be versioned under `extrinsic/v2`, it can be at the top level `types/src`